From: Nathan Rossi <nathan.rossi@xilinx.com>
Subject: [PATCH] xen/x86: Fix up rules when forcing -mno-sse

* When forcing -mno-sse specify -mfpmath=387 to avoid the fallback
  warning

Signed-off-by: Nathan Rossi <nathan.rossi@xilinx.com>
Upstream-Status: Not-Submitted
---
 xen/arch/x86/Rules.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xen/arch/x86/Rules.mk b/xen/arch/x86/Rules.mk
index 6775cb5..03848f7 100644
--- a/xen/arch/x86/Rules.mk
+++ b/xen/arch/x86/Rules.mk
@@ -42,7 +42,7 @@ x86 := y
 x86_32 := n
 x86_64 := y
 
-CFLAGS += -mno-red-zone -mno-sse -fpic
+CFLAGS += -mno-red-zone -mno-sse -fpic -mfpmath=387
 CFLAGS += -fno-asynchronous-unwind-tables
 # -fvisibility=hidden reduces -fpic cost, if it's available
 ifneq ($(call cc-option,$(CC),-fvisibility=hidden,n),n)
-- 
2.1.1

